src/default-config.json

```diff
@@ -7,5 +7,5 @@
   "asn": "enter the ASNs of interest here",
   "peerIp": "enter peer IP",
   "connector": "enter connector or connectors of interests",
-  "windowSize": 360
+  "windowSize": "5"
```

Comment:
I figure this changed only because you're working on things, but the default should probably be higher than this in the final commit